homeassistant/components/nasweb/nasweb_data.py

```diff
@@ -0,0 +1,65 @@
+"""Dataclass storing integration data in hass.data[DOMAIN]."""
+
+from dataclasses import dataclass, field
+import logging
+
+from aiohttp.hdrs import METH_POST
+
+from homeassistant.components.webhook import (
+    async_generate_id,
+    async_register as webhook_register,
+    async_unregister as webhook_unregister,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.network import get_url
+
+from .const import DOMAIN, WEBHOOK_URL
+from .coordinator import NASwebCoordinator, NotificationCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class NASwebData:
+    """Class storing integration data."""
+
+    notify_coordinator: NotificationCoordinator = field(
+        default_factory=NotificationCoordinator
+    )
+    webhook_id = ""
+    entries_coordinators: dict[str, NASwebCoordinator] = field(default_factory=dict)
+
+    def is_initialized(self) -> bool:
+        """Return True if instance was initialized and is ready for use."""
+        return len(self.webhook_id) > 0
```

Comment:
The empty string is falsy, we can just do:
```suggestion
        return bool(self.webhook_id)
```